clients/platform/v0/nodejs/PlatformPromiseClient.js

```diff
@@ -284,6 +290,72 @@ class PlatformPromiseClient {
     );
   }
 
+  /**
```

Comment:
Add this to the constructor

`this.client.getIdentitiesByPublicKeyHashes = promisify(
this.client.getIdentitiesByPublicKeyHashes.bind(this.client),
);`

`this.client.getIdentityIdsByPublicKeyHashes = promisify(
this.client.getIdentityIdsByPublicKeyHashes.bind(this.client),
);`

Reply:
@shuplenkov Sure, thanks!